appinfo/routes.php

```diff
@@ -362,6 +362,33 @@
 				'token' => '^[a-z0-9]{4,30}$',
 			],
 		],
+		[
+			'name' => 'Room#getBridgeOfRoom',
+			'url' => '/api/{apiVersion}/room/{token}/bridge',
+			'verb' => 'GET',
+			'requirements' => [
+				'apiVersion' => 'v(1|2)',
```

Comment:
```suggestion
				'apiVersion' => 'v1',
```

Same below. bridge api is only version 1 for now :)

Reply:
Maybe also split it up a bit, so use:
`/api/{apiVersion}/bridge/{token}` as route and a dedicated `BridgeController`